src/sentry/api/serializers/models/release.py

```diff
@@ -107,11 +108,18 @@ def _get_commit_metadata(self, item_list, user):
             # Increment commit count per release
             commit_count_by_release_id[rc.release_id] += 1
 
+            # look for latest commit by release
+            if latest_commit_by_release_id.get(rc.release_id) is None \
```

Comment:
```python
if rc.release_id not in latest_commit_by_release_id or latest_commit_by_release_id[rc.release_id].order > rc.order
```

Would be a bit more explicit. And avoids doing a `.get()` call twice. Since the first check is just checking if the key exists, and if it does, then we can assume and just use it directly.